client/index.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+const socket = io.connect('http://localhost:3000');
```

Comment:
@blackshady. Hardcoding the URL here means the app will not function properly in other environments that are not `localhost`. I think an environment variable like BASE_URL will work best here so that the URL is dynamic based on the environment.

Reply:
This file was created for anyone who wants to test the real-time functionality. It is not meant to be used anywhere else. After this PR have been tested and approved I will probably take it out.